docs/index.md

```diff
@@ -1,7 +1,15 @@
-<a href="https://islandora.ca">Islandora</a> is an open-source repository framework, that combines a Drupal front-end
-with a Fedora repository. It is a solution for institutions who want an extremely flexible and configurable preservation repository without creating a lot of custom code.
+[Islandora](https://islandora.ca) is an open-source framework that provides the necessary tools turn a [Drupal 8](https://www.drupal.org) website into a fully-functional preservation repository without the need for additional code, while remanining flexible enough to be extended and customized.
 
-Islandora 8 integrates [Drupal](https://www.drupal.org/) (currently, Drupal 8) and Duraspace's [Fedora Repository](https://wiki.duraspace.org/display/FF/Fedora+Repository+Home) project (currently, Fedora 5).
+Out of the box, this includes:
 
-Islandora 8 allows you to create nodes, media, files, and taxonomy terms in Drupal, which are converted to RDF
-and pushed into a Fedora repository. It also allows you to connect repository content with microservices, small webapps that transform files to other types for display or preservation, or to extract additional metadata.
+- A suite of customized Drupal resource types and configurations, packaged as the [Islandora 8 Drupal module](https://github.com/Islandora/islandora), that streamline the process of creating digital preservation resources natively within Drupal as nodes, media, files, and taxonomy terms
```

Comment:
I don't think we should be saying "creating digital preservation resources", since that might suggest that the process of preservation is set-and-forget. I'd much rather see "creating preservable digital resources". It's hair splitting but I think it's an important distinction.

Reply:
Definitely can make that change 👍